tests/h/models/blocklist_test.py

```diff
@@ -1,20 +1,26 @@
-from h import models
+import pytest
 
+from h.models.blocklist import Blocklist
```

Comment:
I like a 1:1 from the unit test to the file under test. We are here to test the `Blocklist` object, we can import it directly.